hack/cma-generate-csv.sh

```diff
@@ -47,7 +47,7 @@ metadata:
+  description: "## About the managed application\\nCustom Metrics Autoscaler for OpenShift is an event driven autoscaler based upon KEDA.  Custom Metrics Autoscaler can monitor event sources like Kafka, RabbitMQ, or cloud event sources and feed the metrics from those sources into the Kubernetes horizontal pod autoscaler.  With Custom Metrics Autoscaler, you can have event driven and serverless scale of deployments within any Kubernetes cluster.\\n## About this Operator\\nThe Custom Metrics Autoscaler Operator deploys and manages installation of KEDA Controller in the cluster. Install this operator and follow installation instructions on how to install Custom Metrics Autoscaler in you cluster.\\n\\n## Prerequisites for enabling this Operator\\n## How to install Custom Metrics Autoscaler in the cluster\\nThe installation of Custom Metrics Autoscaler is triggered by the creation of \`KedaController\` resource. Please refer to the [KedaController Spec](https://github.com/openshift/custom-metrics-autoscaler-operator/blob/main/README.md#the-kedacontroller-custom-resource) for more deatils on available options.\\n\\nOnly resource named \`keda\` in the namespace where the operator is installed will trigger the installation, reconfiguration or removal of the KEDA Controller resource.\\n\\nThere should be only one KEDA Controller in the cluster. \\n\\nThe operator will behave in this manner whether it is installed with the \`AllNamespaces\` or \`OwnNamespace\` install mode. While the operator more closely matches the \`OwnNamespace\` semantics, \`AllNamespaces\` is a supported installation mode to allow it to be installed to namespaces with existing \`OperatorGroups\` which require that installation mode, such as \`openshift-operators\`.\n"
```

Comment:
Nit: I suggest breaking this up into multiple lines if possible. Will make future reviews easier. Otherwise we have to review the whole thing for changes.

Reply:
Fixed, thanks!